yarn.lock

```diff
@@ -4058,6 +4058,10 @@ copy-webpack-plugin@~4.4.1:
     p-limit "^1.0.0"
     serialize-javascript "^1.4.0"
 
+core-js@2.5.5:
+  version "2.5.5"
+  resolved "http://npm.kroger.com/core-js/-/core-js-2.5.5.tgz#b14dde936c640c0579a6b50cabcc132dd6127e3b"
```

Comment:
Looks like this is your private registry and other people can't install from there. You might need to add an empty `.npmrc` to project root (without commiting it), and run `yarn` again

Reply:
Thanks for pointing that out. This has been updated.